go/pkg/username/others.go

```diff
@@ -0,0 +1,16 @@
+// +build !android,!ios
+
+package username
+
+import "os/user"
+
+const defaultUsername = "anon#1337"
+
+func getUsername() string {
+	user, err := user.Current()
+	if err != nil || user == nil {
+		return ""
```

Comment:
```suggestion
		return defaultUsername
```

Reply:
Already handled by the public package function `GetUsername()` in go/pkg/username/username.go:12